package.json

```diff
@@ -2,14 +2,205 @@
+        },
+        "./recipe/thirdpartyemailpassword/emaildelivery": {
+            "import": "./dist/recipe/thirdpartyemailpassword/emaildelivery/services/index.mjs",
+            "types": "./dist/recipe/thirdpartyemailpassword/emaildelivery/services/index.d.ts",
+            "require": "./dist/recipe/thirdpartyemailpassword/emaildelivery/services/index.js"
+        },
+        "./recipe/thirdpartypasswordless": {
+            "import": "./dist/recipe/thirdpartypasswordless/index.mjs",
+            "types": "./dist/recipe/thirdpartypasswordless/index.d.ts",
+            "require": "./dist/recipe/thirdpartypasswordless/index.js"
+        },
+        "./recipe/thirdpartypasswordless/emaildelivery": {
+            "import": "./dist/recipe/thirdpartypasswordless/emaildelivery/services/index.mjs",
+            "types": "./dist/recipe/thirdpartypasswordless/emaildelivery/services/index.d.ts",
+            "require": "./dist/recipe/thirdpartypasswordless/emaildelivery/services/index.js"
+        },
+        "./recipe/thirdpartysmspasswordless/smsdelivery": {
+            "import": "./dist/recipe/thirdpartysmspasswordless/smsdelivery/services/index.mjs",
+            "types": "./dist/recipe/thirdpartysmspasswordless/smsdelivery/services/index.d.ts",
+            "require": "./dist/recipe/thirdpartysmspasswordless/smsdelivery/services/index.js"
+        },
+        "./recipe/usermetadata": {
+            "import": "./dist/recipe/usermetadata/index.mjs",
+            "types": "./dist/recipe/usermetadata/index.d.ts",
+            "require": "./dist/recipe/usermetadata/index.js"
+        },
+        "./recipe/userroles": {
+            "import": "./dist/recipe/userroles/index.mjs",
+            "types": "./dist/recipe/userroles/index.d.ts",
+            "require": "./dist/recipe/userroles/index.js"
+        },
+        "./types": {
+            "import": "./dist/types.mjs",
+            "types": "./dist/types.d.ts",
+            "require": "./dist/types.js"
+        },
+        "./*": "./*"
+    },
+    "module": "./dist/index.js",
+    "main": "./dist/index.mjs",
+    "types": "./dist/index.d.ts",
+    "typesVersions": {
+        "*": {
+            "*": [
+                "*",
+                "dist/*",
+                "dist/*",
+                "dist/*.d.ts",
+                "dist/*/index.d.ts"
+            ]
+        }
+    },
+    "files": [
+        "dist",
+        "LICENSE",
+        "README.md"
+    ],
     "scripts": {
         "test": "TEST_MODE=testing npx mocha --timeout 500000",
-        "build-check": "cd lib && npx tsc -p tsconfig.json --noEmit && cd ../test/with-typescript && npx tsc -p tsconfig.json --noEmit",
-        "build": "cd lib && rm -rf build && npx tsc -p tsconfig.json && cd ../test/with-typescript && npx tsc -p tsconfig.json --noEmit && cd ../.. && npm run post-build",
+        "build-check": "npx tsc -p tsconfig.json --noEmit && cd ../test/with-typescript && npx tsc -p tsconfig.json --noEmit",
+        "build": "tsup",
```

Comment:
the build command earlier used to also check the compilation of `../test/with-typescript` - this was done as a way to make sure that the interface exposed to the user doesn't break unintentionally.

Reply:
you don't need them, tsup sets them all in dist. There is no need for that test, since that user will no longer have ts errors.